data/sponsors/bankinglab.yml

```
@@ -0,0 +1,2 @@
+name: "bankinglab"
+url: "http://bankinglab.com"
```

```suggestion
url: "https://bankinglab.com/"
```

It's best to avoid the HTTP to HTTPS redirect

---

true. uodated now